cli/crates/server/src/bridge/server.rs

```diff
@@ -123,10 +123,16 @@ async fn invoke_resolver_endpoint(
     Json(payload): Json<ResolverInvocation>,
 ) -> Result<Json<serde_json::Value>, ApiError> {
     trace!("resolver invocation\n\n{:#?}\n", payload);
-    super::resolvers::invoke_resolver(handler_state.worker_port, &payload.resolver_name)
-        .await
-        .map_err(|_| ApiError::ResolverInvalid(payload.resolver_name.clone()))
-        .map(Json)
+    super::resolvers::invoke_resolver(
+        handler_state.worker_port,
+        &payload.resolver_name,
+        &serde_json::json!({
+            "parent": payload.payload.parent,
```

Comment:
Can you please pass through the whole `payload.payload`? That includes `args` that for some reason I failed to pass through.